stock_product_ul/models/product_ul.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+import odoo.addons.decimal_precision as dp
+
+
+class ProductUl(models.Model):
```

Comment:
@alexis-via I think post_init script with openupgrade.update_module_moved_fields should be an interesting option as it can simplify migration. What do you think?

Reply:
I can take care of that if you want

Reply:
https://github.com/akretion/stock-logistics-tracking/pull/1